install/yaml/install.yaml

```diff
@@ -860,6 +882,17 @@ spec:
                             type: integer
                             minimum: 1
                             maximum: 65535
+                    loglevel:
```

Comment:
Pretty sure this validation also needs to be changed, with the rename?

Reply:
yes, that's true